src/iterator.rs

```diff
@@ -30,41 +30,13 @@ impl PointLocation {
         match &self {
             PointLocation::AllPoints => Box::new(AllPoints {}),
             PointLocation::Aabb(aabb) => Box::new(aabb.clone()),
-            PointLocation::Frustum(frustum) => Box::new(CachedAxesFrustum::new(frustum.clone())),
-            PointLocation::Obb(obb) => Box::new(CachedAxesObb::new(obb.clone())),
+            PointLocation::Frustum(frustum) => Box::new(frustum.clone()),
+            PointLocation::Obb(obb) => Box::new(obb.clone()),
             PointLocation::S2Cells(cell_union) => Box::new(cell_union.clone()),
         }
     }
 }
 
-/// This macro is an alternative to `get_point_culling()`, to be used where
```

Comment:
why has it been removed?

Reply:
It can be changed to make better use of the traits and be simpler. It's back now :)